billion-scale-image-search/src/main/application/schemas/image.sd

```diff
@@ -77,7 +77,7 @@ schema image {
 
   document-summary vector-summary {
     from-disk
-    summary vector type tensor<bfloat16>(x[768]) {
+    summary vector {
```

Comment:
I suggest removing the `source` specification as well (where applicable) to make the examples as simple as possible. Applies for the other files as well.